cmd/kube-proxy/app/options.go

```diff
@@ -292,9 +294,9 @@ func (o *Options) errorHandler(err error) {
 	o.errCh <- err
 }
 
-// processHostnameOverrideFlag processes hostname-override flag
-func (o *Options) processHostnameOverrideFlag() error {
-	// Check if hostname-override flag is set and use value since configFile always overrides
+// processMaintainedFlags checks if maintained flags are set and, if so, updates the config with
+// the specified value giving precedence to these flags over config file.
+func (o *Options) processMaintainedFlags() error {
```

Comment:
I'm not sure why you are calling these "maintained". I think that may not be the word you're looking for...

Reply:
I was struggling with the name, this is to process flags which will always have precedence over config, unlike the other flags. Will come up with a better name.

Reply:
if for v1alpha2 all flags override config, great.
kubelet and kubeadm do that. kube-proxy was with only some flags overriding.